ipatests/test_integration/test_caless.py

```diff
@@ -328,7 +328,8 @@ def get_replica_filename(self, replica):
                             'replica-info.gpg')
 
     @classmethod
-    def create_pkcs12(cls, nickname, filename='server.p12', password=None):
+    def create_pkcs12(cls, nickname, filename='server.p12', password=None,
+                      no_pin=False):
```

Comment:
You could use the same strategy as in prepare_replica, ie
```
def create_pkcs12(cls, nickname, filename='server.p12', password=_DEFAULT):
```
and 
```
if password is _DEFAULT:
    password = cls.cert_password
[...]
if password:
  # call with -passout...
else:
  # call without -passout
```
This way you don't need to add no_pin argument.